tests/foreman/api/test_syncplan.py

```diff
@@ -193,11 +193,21 @@ def test_positive_create_with_interval(self):
         :CaseImportance: Critical
         """
         for interval in valid_sync_interval():
-            with self.subTest(interval):
+            if interval != SYNC_INTERVAL['custom']:
                 sync_plan = entities.SyncPlan(
-                    interval=interval,
+                     description=gen_string('alpha'),
+                     organization=self.org,
+                     interval=interval
+                ).create()
+
+            elif (interval == 'custom cron'):
+                sync_plan = entities.SyncPlan(
+                    description=gen_string('alpha'),
                     organization=self.org,
+                    interval='custom cron',
```

Comment:
here we should better use the variable `interval=interval,` as above, if the value (custom cron) changes wording in the future, we would need to update it also here

Reply:
OK, commit coming.